### ISSUE TITLE
Banning correct words, such as classroom

### ISSUE THREAD

**Issue by jagermesh** [snipe/banbuilder-library#101]
[orig] => this is our classroom
    [clean] => this is our cl***room

**Comment by lukepass** [comment]
I have the same problem, is there a way to match whole words only? Thanks.

**Comment by lukepass** [comment]
I saw that you can pass a second argument to censorString and it will match whole words only:

```
public function censorString($string, $fullWords = false) // <-- defaults to false, you can put true to match whole words
```

**Comment by jagermesh** [comment]
Yeah, but this will start showing other words which are in 99% cases must be hidden. For example f**k must be considered as bad either as part of other word or as separate word, so regardless of the $fullWords true or false.

**Comment by harpreetsb** [comment]
same problem with word asshole
if we set `fullwords` to `true`,  word `classroom` is passed but then word `asshole` is also passed.

**Comment by rob4226** [comment]
I would still set `fullwords` to `true` then add `asshole` to the dictionary so it will be filtered as a fullword.
